public/css/login.module.css

```diff
@@ -88,7 +119,66 @@
     border-radius: 10px;
     background-color: transparent;
     color: #fff;
-    font-weight: 600d;
+    font-weight: 600;
```

Comment:
if the font family only have one face, the weight will always be consistent. no need for this.